src/main/java/lab2/AvatarGenerator.java

```diff
@@ -1,66 +1,76 @@
+                , "bottts", "croodles", "croodles-neutral", "fun-emoji", "icons", "identicon", "initials", "lorelei",
+                "micah", "miniavs", "open-peeps", "personas", "pixel-art", "pixel-art-neutral" }; // styles is an array of strings
+        // Randomly selects one one avatar style from the array "styles"
+        var style = styles[(int)(Math.random() * styles.length)]; // "style" is a String
 
-        // Generate a random seed
-        var seed = (int)(Math.random() * 10000);
+        // Generates the random seed for the avatar HTTP request
+        var seed = (int)(Math.random() * 10000); // 'seed' is a primitive type, integer value
 
-        // Create an HTTP request for a random avatar
+        // Create an HTTP request URI to fetch a random avatar with a seed
         var uri = URI.create("https://api.dicebear.com/9.x/%s/png?seed=%d".formatted(style, seed));
+        // uri returns a java.net.URI type, reference type
+
+        // Create an HttpRequest object with the generated URI
         var request = HttpRequest.newBuilder(uri).build();
+        // request returns a java.net.http.HttpRequest type, reference type
 
-        // Send the request
-        try (var client = HttpClient.newHttpClient()) {
-            var response = client.send(request, HttpResponse.BodyHandlers.ofInputStream());
-            return response.body();
+        // send the HTTP request and get the avatar image as InputStream
+        try (var client = HttpClient.newHttpClient()) { // create a new HttpClient instance to send the HTTP request
+            var response = client.send(request, HttpResponse.BodyHandlers.ofInputStream());// sends the request and receives the response as an InputStream
+            //repose returns a java.net.http.HttpResponse<java.io.InputStream>, reference type
+            return response.body(); // returns the response body with InputStream containing the avatar image
         }
     }
 
-    public static void showAvatar(InputStream imageStream) {
-            JFrame frame = new JFrame("PNG Viewer");
-            frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-            frame.setResizable(false);
-            frame.setSize(200, 200);
-            frame.getContentPane().setBackground(Color.BLACK);
+    public static void showAvatar(InputStream imageStream) { // 'imageStream' is an InputStream parameter, so it is a reference type
+        JFrame frame = new JFrame("PNG Viewer"); // Creates JFrame object with a title "PNG Viewer"
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE); // Sets the default close operation for the frame (exit the application)
+        frame.setResizable(false); // Makes the frame non-resizable
+        frame.setSize(200, 200); // Sets the window size to 200x200 pixels
+        frame.getContentPane().setBackground(Color.BLACK); // Sets the background color of thw window to black
```

Comment:
EXIT_ON_CLOSE and BLACK are class variables.  All the arguments here are primitive values